source/references.rst

```diff
@@ -0,0 +1,6 @@
+.. SPDX-License-Identifier: Apache-2.0
+
+References
+==========
+
+.. [dtspec]
```

Comment:
do you want to add the dtspec link here?
https://www.devicetree.org/specifications/

Reply:
added